CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # bedrock-vue-organization-form ChangeLog
 
-## 1.1.0 - xxxxxx
+## 2.0.0 -
 
 ### Added
-- Added new entity types.
+- Add new entity types.
+- Add Eslint and dependencies to the project.
+
+### Removed
+- Deprecate components such as q-layout-header.
+- Deprecate css classes such as gutter-md.
```

Comment:
If these are only deprecated and not fully removed, let's put it under `Changed`

Reply:
their names were changed actually.

Reply:
Then they should be under `Changed` and say ```***BREAKING***: Changed X to Y.``` for each one.

Reply:
I changed it - Remove deprecated components such as q-layout-header.

Reply:
Ok, that makes more sense! +1

Reply:
lol then I changed to - **BREAKING** anyways what's there is ok.